### PR TITLE
#163581946 build candidate application page 

### PR DESCRIPTION
**Pull request by codeBlock-1984**
**What does this PR do?**
Build a page for users to apply for a political office

**Description of Tasks to be completed**
- Create page layout
- Build a candidate application form with all relevant input fields
- Create buttons for submitting an application, and canceling "application" action
- Write the functions to implement user application

**How should it be tested?**
- On any page, click on the "contest" link on the left side-navigation panel.
- Fill out the contest form on the page
- Click on the "submit" button to submit application

### CODE REVIEW

**Comment by ascii-dev** [comment]
My comments here https://github.com/codeBlock-1984/Politico/pull/1#issuecomment-459125157 also apply here.

File: contest.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+                      <p>This is the beta version of a test product of ICE Inc.</p>
+        
+                </div>
+           </div>
+      </div> 
+   
+   <script src="js/script.js"></script>   
+  </body>
+
+  </html>
```
Comment (ascii-dev) [inline contest.html]: A new line here, please

File: js/script.js [hunk 2]
```diff
@@ -96,3 +96,16 @@ function displayMyVote(){
...
+
+let clearForm = () => {
+  document.getElementById("contest-form").reset();
+}
+let conSubmit = () => {
+  let fName = document.getElementById("fN").value;
+  let lName = document.getElementById("lN").value;
+  let party = document.getElementById("pty").value;
+  let office = document.getElementById("of").value;
+  let manifesto = document.getElementById("man").value;
```
Comment (ascii-dev) [inline js/script.js]: I think your `ids` should be descriptive. Please use descriptive names in your code. Thanks

Comment (codeBlock-1984) [reply js/script.js]: Okay, I will make the changes

File: contest.html (hunk 1 above)
Comment (codeBlock-1984) [reply contest.html]: Thanks for pointing that out. I will effect the change ASAP.

**Pull request merged**